test/verifiedTests/base/testIO/testConvertCobraLP2mps.m

```diff
@@ -0,0 +1,30 @@
+% The COBRAToolbox: testConvertCobraLP2mps.m
+%
+% Purpose:
+%     - test the convertCobraLP2mps function
+%
+% Authors:
+%     - Jacek Wachowiak
+global CBTDIR
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testConvertCobraLP2mps'));
+cd(fileDir);
+
+% test variables
+LPProblem = readCbModel([CBTDIR filesep 'test' filesep 'models' filesep 'ecoli_core_model.mat']);
+LPProblem_2 = rmfield(LPProblem, 'csense');
```

Comment:
Why are the `osense` and `csense` fields removed here?

Reply:
The function checks if they exist and then if not, it creates them with a default argument